functions/10_gutenberg.php

```diff
@@ -0,0 +1,119 @@
+<?php
+/**
+ * 使用できるブロックタイプを制限
+ */
+add_filter('allowed_block_types', 'block_types_white_list');
+function block_types_white_list($allowed_block_types)
+{
+    $allowed_block_types = array(
+        // 一般ブロック
+        'core/paragraph', // 段落
+        'core/heading', // 見出し
+        'core/image', // 画像
+        'core/quote', // 引用
+//        'core/gallery', // ギャラリー
+        'core/list', // リスト
+//        'core/audio', // 音声
+//        'core/cover', // カバー
+//        'core/file', // ファイル
+//        'core/video', // 動画
+
+        // フォーマット
+//        'core/code', // ソースコード
+//        'core/freeform', // クラシック
+//        'core/html', // カスタムHTML
+//        'core/preformatted', // 整形済み
+//        'core/pullquote', // プルクオート
+//        'core/table', // テーブル
+//        'core/verse', // 詩
+
+        // レイアウト要素
+//        'core/button', // ボタン
+//        'core/columns', // カラム
+//        'core/media-text', // メディアと文章
+//        'core/more', // 続きを読む
+//        'core/nextpage', // 改ページ
+//        'core/separator', // 区切り
+//        'core/spacer', // スペーサー
+
+        // ウィジェット
+//        'core/shortcode', // ショートコード
+//        'core/archives', // アーカイブ
+//        'core/categories', // カテゴリー
+//        'core/latest-comments', // 最新のコメント
+//        'core/latest-posts', // 最新の記事
+
+        // 埋め込み
+//        'core/embed', // 埋め込み
+//        'core-embed/twitter', // Twitter
+//        'core-embed/youtube', // YouTube
+//        'core-embed/facebook', // Facebook
+//        'core-embed/instagram', // Instagram
+//        'core-embed/wordpress', // WordPress
+//        'core-embed/soundcloud', // SoundCloud
+//        'core-embed/spotify', // Spotify
+//        'core-embed/flickr', // Flickr
+//        'core-embed/vimeo', // Viemo
```

Comment:
```suggestion
//        'core-embed/vimeo', // Vimeo
```
こうしたらここでコミットできるきがする（ためしたい）